cmd/mesh/manifest-versions_test.go

```diff
@@ -113,9 +118,8 @@ func TestGetVersionCompatibleMap(t *testing.T) {
 				binVersion:  goVerNonexistent,
 				l:           l,
 			},
-			want: nil,
-			wantErr: fmt.Errorf("this operator version %s was not found in the version map",
-				goVerNonexistent.String()),
+			want:    latestCm,
+			wantErr: nil,
```

Comment:
There's a logic problem here. If we specify a version like 0.1, we don't want to return the latest version but rather return an error.

Reply:
fixed, thanks